apps/page/option/banned_word.ts

```diff
@@ -64,6 +66,36 @@ export default class BannedWords {
     });
   }
 
+  private assertButtonElement(element: HTMLElement | null): HTMLButtonElement {
+    if (!element) {
+      throw new Error("bannedWordAddButton is null");
+    }
+    if (element instanceof HTMLButtonElement) {
+      return element as HTMLButtonElement;
```

Comment:
`as HTMLButtonElement` is redundant.
The same applies to the other two locations.